service/build.gradle

```diff
@@ -50,74 +51,88 @@ repositories {
+    }
 
-    // [Application DB] MySQL/MariaDB
-    implementation group: "org.mariadb.jdbc", name: "mariadb-java-client"
-    implementation group: "mysql", name: "mysql-connector-java", version: "8.0.30"
-    implementation group: "com.google.cloud.sql", name: "mysql-socket-factory", version: "1.7.0"
-
-    // GCP libraries. Most versions are controlled by the BOM specified in buildSrc.
-    implementation group: "com.google.cloud", name: "google-cloud-bigquery"
-    implementation group: "com.google.cloud", name: "google-cloud-storage"
-    implementation group: "com.google.guava", name: "guava"
-    implementation group: "com.google.apis", name: "google-api-services-oauth2", version: "v2-rev20200213-2.0.0"
-
-    // OpenApi/Swagger
-    implementation group: "com.fasterxml.jackson.dataformat", name: "jackson-dataformat-yaml", version: "2.11.2"
-    implementation group: "com.fasterxml.jackson.module", name: "jackson-module-jaxb-annotations", version: "2.11.2"
-    implementation group: "com.fasterxml.jackson.core", name: "jackson-core", version: "2.12.3"
-    implementation group: "com.fasterxml.jackson.datatype", name: "jackson-datatype-jsr310", version: "2.12.3"
-    implementation group: "io.swagger.core.v3", name: "swagger-annotations"
-    implementation group: "io.swagger", name: "swagger-annotations", version: "1.6.2"
-    implementation group: "org.webjars", name: "webjars-locator-core", version: "0.48"
-    runtimeOnly group: "org.webjars.npm", name: "swagger-ui-dist", version: "3.36.2"
-    swaggerCodegen group: "io.swagger.codegen.v3", name: "swagger-codegen-cli"
+    implementation "org.apache.commons:commons-text:${gradle.vApacheCommonsText}"
+    implementation 'org.apache.commons:commons-dbcp2'
+    implementation 'org.apache.commons:commons-pool2'
 
     // Spring version controlled dependencies
-    annotationProcessor group: "org.springframework.boot", name: "spring-boot-configuration-processor"
-    implementation group: "javax.validation", name: "validation-api"
-    implementation group: "org.apache.commons", name: "commons-dbcp2"
-    implementation group: "org.apache.commons", name: "commons-pool2"
-    implementation group: "org.springframework.boot", name: "spring-boot-starter-web"
-    implementation group: "org.springframework.boot", name: "spring-boot-starter-data-jdbc"
-    implementation group: "org.springframework.security", name: "spring-security-core"
-
-    // Spring version controlled JUnit dependencies.
+    annotationProcessor 'org.springframework.boot:spring-boot-configuration-processor'
+    implementation 'org.springframework.boot:spring-boot-starter-data-jdbc'
+    implementation 'org.springframework.boot:spring-boot-starter-web'
+    implementation 'org.springframework.security:spring-security-core'
+    implementation 'jakarta.validation:jakarta.validation-api'
+    implementation 'jakarta.servlet:jakarta.servlet-api:6.0.0'
+    implementation 'jakarta.ws.rs:jakarta.ws.rs-api:4.0.0'
```

Comment:
Comment says Spring controlled dependencies -- should these have a version specified inline?

Reply:
Split them into separate blocks since these need specific versions. Same in other build files